network-store-model/src/main/java/com/powsybl/network/store/model/ActivePowerControlAttributes.java

```diff
@@ -29,4 +29,8 @@ public class ActivePowerControlAttributes implements ExtensionAttributes {
     private double droop;
 
     private double participationFactor;
+
+    private Double minTargetP;
```

Comment:
Could you change min/maxTargetP into double ?

Reply:
Changed